tests/integration/integration-tests.yml

```diff
@@ -4018,7 +4018,7 @@
       assert:
         that:
           - test_three is changed
-          - test_three['diff']['after']['upstream_speed'] == 1000
+
```

Comment:
Can you add this line back? After that I will get this merged.

Thanks for all your work on this! I will try and update the documentation as well to help understand what we're looking for when contributing new modules, etc.